pkg/apis/core/v1alpha1/federatedtypeconfig_types.go

```diff
@@ -119,11 +119,13 @@ func setStringDefault(value *string, defaultValue string) {
 	*value = defaultValue
 }
 
-// PluralNameForKind naively computes the plural name from the kind by
-// lowercasing and suffixing with 's'.
-func pluralName(kind string) string {
+// PluralName computes the plural name from the kind by
+// lowercasing and suffixing with 's' or `es`.
+func PluralName(kind string) string {
 	lowerKind := strings.ToLower(kind)
-	if strings.HasSuffix(lowerKind, "s") {
+	if strings.HasSuffix(lowerKind, "s") || strings.HasSuffix(lowerKind, "x") ||
```

Comment:
Is this list of suffixes inspired by a list written by people who have thought a lot of about pluralizing? (I have not :))

Reply:
Whoops, missed the link in your PR description.